Wikipedia/Code/StorageAndSyncingSettingsViewController.swift

```diff
@@ -0,0 +1,213 @@
+struct SettingsSection {
+    let headerTitle: String?
+    let footerText: String?
+    let items: [SettingsItem]
+}
+
+struct SettingsItem {
+    let disclosureType: WMFSettingsMenuItemDisclosureType?
+    let itemType: StorageAndSyncingSettingsItemType
+    let title: String?
+    let iconName: String?
```

Comment:
Looks like `iconName` is never not `nil` so can be removed from Item and just pass nil to WMFSettingsMenuItemDisclosureType's configure method (`iconColor` and `iconBackgroundColor` too)